### PR TITLE
EN-1385-file-browser-create-folder

### PR DESCRIPTION
**Pull request by alepintus**
Hi @alessioscalici @nicpuddu @fedelocci @mcarta-rt ,
File browser create folder page added.

Could you please have a look?

Thanks!

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/ui/file-browser/add/CreateFolderForm.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+CreateFolderFormBody.propTypes = {
+  handleSubmit: PropTypes.func,
+  invalid: PropTypes.bool,
+  submitting: PropTypes.bool,
+  onWillMount: PropTypes.func,
+};
+
+CreateFolderFormBody.defaultProps = {
+  handleSubmit: () => {},
```
Comment (nicpuddu) [inline src/ui/file-browser/add/CreateFolderForm.js]: handle submit is always injected, there is no need for a default

File: src/ui/file-browser/add/CreateFolderForm.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  invalid: false,
+  submitting: false,
+  onWillMount: () => {},
```
Comment (nicpuddu) [inline src/ui/file-browser/add/CreateFolderForm.js]: same applies here: it should just be required

**Comment by entando-service** [comment]
All is well

**Pull request merged**